plugins/main/public/controllers/register-agent/components/server-address/server-address.tsx

```diff
@@ -42,6 +46,50 @@ const ServerAddressInput = (props: ServerAddressInputProps) => {
       isPopoverServerAddress => !isPopoverServerAddress,
     );
   const closeServerAddress = () => setIsPopoverServerAddress(false);
+  const [rememberServerAddress, setRememberServerAddress] = useState(false);
+  const [defaultServerAddress, setDefaultServerAddress] = useState(formField.initialValue ? formField.initialValue : '')
```

Comment:
`formField` can be `undefined`
Idem in other lines

Reply:
Fixed in [9d04786](https://github.com/wazuh/wazuh-kibana-app/pull/5791/commits/9d047862f9dc464d091a118427379b1994a45478)